code/gamehelp/gameplayhelp.cpp

```diff
@@ -22,33 +23,41 @@
 
 
 
-// text positioning constats
+// text positioning constants
 #define TITLE_Y			35
 #define Y_START			70
 #define X_OFFSET_1		70
 
 // pixel offset for description of key from where key binding starts
-#define KEY_DESCRIPTION_OFFSET	193	
+// increased this from 193 to 250 to account for allowing multiple keys -Mjn
+#define KEY_DESCRIPTION_OFFSET	250	
 
 // different gameplay help pages
-#define GP_FIRST_SCREEN									0		// keep up to date
-
-#define GP_HELP_BASIC_KEYS								0
-#define GP_HELP_MOVEMENT_KEYS							1
-#define GP_HELP_COMMON_TARGET_KEYS					2
-#define GP_HELP_ADVANCED_TARGET_KEYS				3
-#define GP_HELP_MESSAGING								4
-#define GP_HELP_WEAPON_KEYS							5
-#define GP_HELP_THROTTLE_AND_ETS_KEYS				6
-#define GP_HELP_VIEW_KEYS								7
-#define GP_HELP_MISC_KEYS								8
-#define GP_HELP_MULTI_KEYS								9
+enum : int {
+	// The first 9 are the retail pages
+	GP_HELP_BASIC_KEYS,
+	GP_HELP_MOVEMENT_KEYS,
+	GP_HELP_COMMON_TARGET_KEYS,
+	GP_HELP_ADVANCED_TARGET_KEYS,
+	GP_HELP_MESSAGING,
+	GP_HELP_WEAPON_KEYS,
+	GP_HELP_THROTTLE_AND_ETS_KEYS,
+	GP_HELP_VIEW_KEYS,
+	GP_HELP_MISC_KEYS,
+	GP_HELP_MULTI_KEYS,
+
+	// Any others defined here will have unique sections within the vector
+	// but will need special handling to display in the retail UI
+	GP_HELP_FUNCTION_KEYS
+};
 
+#define GP_FIRST_SCREEN									0		// keep up to date
 #define GP_LAST_SCREEN_SINGLE							8		// keep up to date
 #define GP_LAST_SCREEN_MULTI							9		// keep up to date
+#define GP_NUM_HELP_SECTIONS							10		// keep up to date
```

Comment:
`NUM_` typically refers to the actual number, not the maximum index.  This should be changed.  You could also put it as the last entry in the enum list, similar to ship flags and parse flags, to have the compiler do this automatically for you.

Reply:
Changed.